### DISCUSSION TITLE
global_normalization vs global_time_normalization

### DISCUSSION THREAD

**Discussion by Alan-Wolf-VN** [ourownstory/neural_prophet#1238] [Q&A - get help using NeuralProphet] [answered]
Can anyone explain what are global_normalization and global_time_normalization?

I try to build a global local model which forecasts many time series that share "some behaviour". However, the default option: global_normalization = False leads to the following error when I run m.fit:

ValueError: Dataset name '__df__' missing from training data params. Set unknown_data_normalization to use global (average) normalization parameters

How to avoid this error? Why setting global_normalization = True does not lead to this error?

**Comment by LeonieFreisinger** [comment, marked as answer]
@Alan-Wolf-VN 
Welcome to NP!
Can you please share all the model parameters you selected? Did you set bot parameters `global_normalization `and `global_time_normalization ` to `False`?
